### ISSUE TITLE
Unable to get RTX 30 series GPUs

### ISSUE THREAD

**Issue by jnstockley** [JonathanVusich/pcpartpicker#49]
When I set up the API and have it retrieve video-card none of the new RTX 30 series show up.

```
api = API()
api.set_region('us')

data = api.retrieve("video-card")

for part in data.values():
    for blah in part:
        print(blah.chipset
```

**Comment by JonathanVusich** [comment]
Yeah, I haven't done a database refresh in a while. I need to set up some kind of automatic build but I don't have the bandwidth right now. I can look into refreshing the database.

**Comment by mfeinst** [comment]
Seconded - a refresh would be awesome and much appreciated! I was under the impression that it will pull whatever data is on the site - is there another "middle" layer involved? Would love to know more about this. I also noticed another very recent issue for the video-cards and memory subcategories that causes unhandled exceptions. Specifically:

**_video-cards: missing value for field "interface"
memory: missing value for field "form_factor"_**

Here is the full error output for the video-cards exception:

> 
> Traceback (most recent call last):
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/dacite/core.py", line 55, in from_dict
>     field_data = data[field.name]
> KeyError: 'interface'
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/dacite/core.py", line 67, in from_dict
>     value = get_default_value_for_field(field)
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/dacite/dataclasses.py", line 21, in get_default_value_for_field
>     raise DefaultValueNotFoundError()
> dacite.dataclasses.DefaultValueNotFoundError
> 
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File “script.py", line 21, in <module>
>     data = api.retrieve(cat)
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/pcpartpicker/api.py", line 55, in retrieve
>     return self._handler.retrieve(*args, force_refresh=force_refresh)
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/pcpartpicker/handler.py", line 96, in retrieve
>     parsed_data = parse(raw_data)
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/pcpartpicker/parse_utils.py", line 34, in parse
>     results = [deserialize_part_data(item) for item in part_dict.items()]
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/pcpartpicker/parse_utils.py", line 34, in <listcomp>
>     results = [deserialize_part_data(item) for item in part_dict.items()]
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/pcpartpicker/parse_utils.py", line 30, in deserialize_part_data
>     return [dataclass_from_dict(part_classes[part_data[0]], item) for item in deserialized_parts]
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/pcpartpicker/parse_utils.py", line 30, in <listcomp>
>     return [dataclass_from_dict(part_classes[part_data[0]], item) for item in deserialized_parts]
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/pcpartpicker/parse_utils.py", line 23, in dataclass_from_dict
>     dataclass = from_dict(datatype, result, config=Config(check_types=False))
>   File "/Users/mfeinst/.pyenv/versions/3.8.0/lib/python3.8/site-packages/dacite/core.py", line 71, in from_dict
>     raise MissingValueError(field.name)
> dacite.exceptions.MissingValueError: missing value for field "interface"



Thanks again for an awesome API!
Michael

**Comment by JonathanVusich** [comment]
Originally I did use that concept but the problem was that whenever there was a website update the entire API broke. Since pc parts don't change that quickly, I just periodically do a scan of the website and then cache the results. Right now it is broken because I am in the process of verifying the new database and trying to fix some broken CI (urgh). I'll try to have it up and running by the end of today.

**Comment by JonathanVusich** [comment]
These issues should now all be addressed with v2.2.2.

**Comment by mfeinst** [comment]
Thank you!
